docs/dev.rst

```diff
@@ -884,9 +886,14 @@ Let's process crashes for Firefox from yesterday. We'd do this:
   # Copy that data from the host into the localstack s3 container
   app@socorro:/app$ bin/socorro_aws_s3.sh sync ./crashdata s3://dev-bucket/
 
-  # Add all the crash ids to the queue
+  # if using CLOUD_PROVIDER=AWS (default)
+  # Add all the crash ids to the sqs queue
   app@socorro:/app$ cat crashids.txt | socorro-cmd sqs publish local-dev-standard
 
+  # or if using CLOUD_PROVIDER=GCP
+  # Add all the crash ids to the pubsub topic
+  app@socorro:/app$ cat crashids.txt | socorro-cmd pubsub publish local-standard-topic
```

Comment:
```suggestion
  app@socorro:/app$ cat crashids.txt | socorro-cmd pubsub publish test local-standard-topic
```

As a side note, "test" is the project id set in both `docker/config/local_dev.env` and `docker/config/test.env`. Maybe we should change the local dev one to "local"? If we did that, we'd use "local" here, too.

Reply:
Typically we try to keep local dev and test environments separate. Would it affect local dev state when we run the tests which include setup/teardown if both have the same project id?

Reply:
> Would it affect local dev state when we run the tests which include setup/teardown if both have the same project id?

no. the only thing we do with pubsub that has a broader impact than specific topics is restarting the container which drops everything from both dev and test